__tests__/app.test.js

```diff
@@ -331,3 +331,54 @@ describe.only("GET /api/users", () => {
       });
   });
 });
+
+describe.only("GET /api/articles (filter)", () => {
+  test("200: Returns an array of articles, sorted by created_at in descending order by default", () => {
```

Comment:
A good test but this looks like exactly the one in our original /api/articles 200 test 👀